### PR TITLE
*: add logging

### LINKED ISSUE #217: Add logs
Add logs to gitbase, to be able to see what is happening on execution, and improve error reporting.

### PR DESCRIPTION
**Pull request by erizocosmico**
Closes #217

### CODE REVIEW

**Comment by erizocosmico** [comment]
Sure @ajnavarro

File: cmd/gitbase/server.go [hunk 1]
```diff
@@ -85,5 +95,7 @@ func (c *CmdServer) Execute(args []string) error {
 		return err
 	}
 
+	logrus.Debug("starting server")
```
Comment (ajnavarro) [inline cmd/gitbase/server.go]: maybe we should add a defer logrus.Debug("server started")

Comment (erizocosmico) [reply cmd/gitbase/server.go]: Start never returns until the connection stops, so a defer would actually be "stopping server" 😛

Comment (ajnavarro) [reply cmd/gitbase/server.go]: oh you're right

**Pull request merged**